### PR TITLE
DiskShadow Execution of script

### PR DESCRIPTION
**Pull request by ljstella**
**Details:**
Added another simple method for T1218, utilizing DiskShadow.exe's Script mode (the -s or /S flag) as outlined [here](https://bohops.com/2018/03/26/diskshadow-the-return-of-vss-evasion-persistence-and-active-directory-database-extraction/)
 
**Testing:**
Executed test Windows Server 2016 using Invoke-AtomicTest

**Associated Issues:**
N/A

### CODE REVIEW

File: atomics/T1218/T1218.yaml [hunk 1]
```diff
@@ -223,3 +223,30 @@ atomic_tests:
   executor:
     command: 'Invoke-ATHRemoteFXvGPUDisablementCommand -ModuleName #{module_name} -ModulePath #{module_path}'
     name: powershell
+- name: DiskShadow Command Execution
+  description: |
+    Emulates attack with a DiskShadow.exe being used to execute arbitrary commands
```
Comment (clr2of8) [inline atomics/T1218/T1218.yaml:228]: Could you include the link to the blog post that you have in the Pull Request description, it is excellent and a good reference for everyone running this atomic. Thx

File: atomics/T1218/T1218.yaml [hunk 2]
```diff
@@ -223,3 +223,30 @@ atomic_tests:
...
+      type: Path
+      default: C:\Windows\System32\diskshadow.exe
+  dependency_executor_name: powershell 
+  dependencies: 
+    - description: txt file must exist on disk at specified location (#{txt_payload})
+      prereq_command: |
+        if (Test-Path #{txt_payload}) {exit 0} else {exit 1}
+      get_prereq_command: |
+        New-Item -Type Directory (split-path #{txt_payload}) -ErrorAction ignore | Out-Null
+        Invoke-WebRequest "https://github.com/redcanaryco/atomic-red-team/raw/master/atomics/T1218/src/T1218.txt" -OutFile "#{txt_payload}"
```
Comment (clr2of8) [inline atomics/T1218/T1218.yaml:247]: Could you add a dependency that will check that diskshadow.exe exists at `dspath`? You don't necessarily need to automate the get_prereq part, you could just print a msg that no automated steps to install are provided. Also, can you make a note somewhere that diskshadow is included by default in Windows Server?

**Review by clr2of8 — CHANGES_REQUESTED** [review]
Excellent first time contribution. I loved reading the blog post and learning more about this. Just a few small requests for change and then it should be good to go. If you want to discuss interactively, you can find me on the Red Canary Slack workspace as @OrOneEqualsOne

**Comment by clr2of8** [comment]
It would be great to provide another Pull Request using this method to dump the ntds.dit file under T1003.003

File: atomics/T1218/T1218.yaml (hunk 1 above)
Comment (ljstella) [reply atomics/T1218/T1218.yaml]: Thanks for the feedback, I've expanded the description to include a reference with the link from the PR.

File: atomics/T1218/T1218.yaml (hunk 2 above)
Comment (ljstella) [reply atomics/T1218/T1218.yaml:247]: Thanks for the feedback, I've added a second prereq test to verify the existence of DiskShadow.exe at its default path, and added wording to the description to clarify that this is a Microsoft shipped-by-default binary in Windows Server.

**Review by clr2of8 — APPROVED** [review]

**Comment by clr2of8** [comment]
Excellent! Keep it up and don´t forget to [claim your free t-shirt](https://github.com/redcanaryco/atomic-red-team/wiki/Contributing#claim-your-free-t-shirt) for being a contributor.

**Pull request merged**